src/main/java/org/betonquest/betonquest/utils/math/Operator.java

```diff
@@ -14,7 +14,7 @@
 public enum Operator {
 
     /**
-     * <b>+</b> Operator, adds two values
+     * <b>+</b> Operator, adds two values.
```

Comment:
Technically all of these short descriptions are short phrases, not grammatically correct sentences. 
So putting punctuation marks behind it is really irritating to me as son of a German teacher.

Nevertheless I get that it could be beneficial for consistency.
But then please make it consistent everywhere. This spot has been missed:

https://github.com/BetonQuest/BetonQuest/blob/a3fc7a0cfcf6a748852b0054e8e17f93ae0f8805/src/main/java/org/betonquest/betonquest/utils/math/Tokenizer.java#L67

Reply:
Thanks for spotting that. The period is the delimiter for the short descriptions shown in the overviews, so it isn't a grammatical but a Javadoc thing.